schema/08-gov.sql

```diff
@@ -1,15 +1,15 @@
 CREATE TABLE proposal
 (
-    title             TEXT    NOT NULL,
-    description       TEXT    NOT NULL,
-    proposal_route    TEXT    NOT NULL,
-    proposal_type     TEXT    NOT NULL,
-    proposal_id       DECIMAL NOT NULL PRIMARY KEY,
-    submit_time       TIMESTAMP,
-    deposit_end_time  TIMESTAMP,
-    voting_start_time TIMESTAMP,
-    voting_end_time   TIMESTAMP,
```

Comment:
Sometime `deposit_end_time`,`voting_start_time`,and `voting_end_time` is null when voting is not start yet, isn't it?

Reply:
Ok, didn't know that. Fixed :100: